PSReadLine/Movement.vi.multiline.cs

```diff
@@ -12,7 +12,8 @@ public partial class PSConsoleReadLine
         /// <param name="arg" />
         public void MoveToFirstLine(ConsoleKeyInfo? key = null, object arg = null)
         {
-            if (!LineIsMultiLine())
+            var count = GetLogicalLineCount();
+            if (count == 1)

```

Comment:
`count` is not used anywhere in this method. In this case, `!LineIsMultiLine()` is better as it's less expensive when it is a mutliple-line input.

Reply:
Even though there will still be two references to `LineIsMultiline()` I’m trying to get rid of this method altogether.
Would a code like this be OK for you:

```
if (GetLogicalLineCount() == 1)
{
    Ding(key, arg);
    return;
}
```

Reply:
Done. Although I liked the symmetry of both methods and considered extracting the common portion to a separate common implementation method.